README.md

```diff
@@ -102,13 +102,13 @@ Output:
 
 Running with docker:
 
-```
+```powershell
```

Comment:
I thinks this one is not PowerShell friendly. It is for cmd/bash

Reply:
Oops. Fixed.